cmake/modules/SwiftPMConfig.cmake.in

```diff
@@ -0,0 +1,4 @@
+
+if(NOT TARGET Build)
```

Comment:
why this specific target?

Reply:
I'm happy to change it to any other target, thats what I had in my buffer at the time :-p